aten/src/ATen/TensorImpl.cpp

```diff
@@ -41,6 +42,23 @@ void Tensor::backward(
   pImpl->backward(std::move(gradient), keep_graph, create_graph);
 }
 
+TensorImpl::TensorImpl(Backend backend, ScalarType scalar_type) {
+  type_ = &globalContext().getType(backend, scalar_type);
+  Storage* storage = type_->storage().release();
+  StorageImpl* storage_impl = storage->pImpl();
+  // storage_impl->retain();
```

Comment:
What's this?

Reply:
Commented it out because I did that at first, but then caused a memory leak. I consider this to be somewhat work in progress, hence the offline direct review request.

Reply:
the retain doesn't seem necessary because the impl should have refcoutn 1.